applications/SwimmingDEMApplication/tests/tests_python_scripts/candelier_scripts/candelier.py

```diff
@@ -1,10 +1,7 @@
 import math
-import cmath
-import mpmath
+import scipy.special
 import numpy as np
```

Comment:
neither scipy nor numpy are part of the python standard library, hence if you use this in a test and they are not available, the test will fail

Check [this](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CoSimulationApplication/tests/test_CoSimulationApplication.py#L7-L11) or [this](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CoSimulationApplication/tests/co_simulation_test_factory.py#L127-L128) to see how I solve those problems, but it can be done in different ways too.

Reply:
Sure, I can do that.

Reply:
this is how it looks like otherwise (output of the Win CI)

~~~
Running SwimmingDEMApplication tests
Traceback (most recent call last):
  File "d:\a\Kratos\Kratos\bin\Custom\applications//SwimmingDEMApplication/tests/test_SwimmingDEMApplication.py", line 11, in <module>
    import NightTests
  File "d:\a\Kratos\Kratos\bin\Custom\applications\SwimmingDEMApplication\tests\NightTests.py", line 10, in <module>
    import TestFactory as TF
  File "d:\a\Kratos\Kratos\bin\Custom\applications\SwimmingDEMApplication\tests\TestFactory.py", line 9, in <module>
    from tests_python_scripts.candelier_scripts.candelier_analysis import CandelierBenchmarkAnalysis
  File "d:\a\Kratos\Kratos\bin\Custom\applications\SwimmingDEMApplication\tests\tests_python_scripts\candelier_scripts\candelier_analysis.py", line 10, in <module>
    import candelier
  File "d:\a\Kratos\Kratos\bin\Custom\applications\SwimmingDEMApplication\tests\tests_python_scripts\candelier_scripts\candelier.py", line 2, in <module>
    import scipy.special
ModuleNotFoundError: No module named 'scipy'
d:\a\Kratos\Kratos\bin\Custom\applications//SwimmingDEMApplication/tests/test_SwimmingDEMApplication.py
~~~